app/models/pipeline.rb

```diff
@@ -10,22 +10,18 @@ class Pipeline < ApplicationRecord # rubocop:todo Metrics/ClassLength
   include Pipeline::BatchValidation
   include SharedBehaviour::Named
 
-  ALWAYS_SHOW_RELEASE_ACTIONS = false # Override this in subclasses if you want to display action links for released batches
-
   # Rails class attributes
   self.inheritance_column = 'sti_type'
 
   # Custom class attributes
   class_attribute :batch_worksheet,
                   :requires_position,
                   :inbox_partial,
-                  :library_creation,
```

Comment:
These attributes are now the same for all piplines, so I've managed to remove them and simplify the associated behaviour.